go.mod

```diff
@@ -49,6 +48,7 @@ require (
 	github.com/mattn/go-zglob v0.0.2
 	github.com/montanaflynn/stats v0.6.3
 	github.com/openhistogram/circonusllhist v0.3.1-0.20210608220433-1bd1bfa6c998
+	github.com/openshift-eng/openshift-goimports v0.0.0-20220201193023-4f8ea117352c // indirect
```

Comment:
@coreydaley please also update the underscoe import in `hack/tools.go` to use the new location

Reply:
Fixed, thanks!